### PR TITLE
rename sosemanuk_state *ss > sosemanuk_state *st

### PR DESCRIPTION
**Pull request by buggywhip**
for consistency across all stream ciphers, rename sosemanuk_state *ss to sosemanuk_state *st  for the yet unreleased Sosemanuk

### Checklist

* [x] documentation is added or updated
* [x] tests are added or updated

### CODE REVIEW

**Comment by karel-m** [comment]
Larry, did you used the latest develop as the base for this PR?

**Comment by buggywhip** [comment]
yes and I noticed someone had already performed the name change on sober128_state.  Thank you!

**Comment by sjaeckel** [comment]
rebase please, not merging back :)

**Comment by buggywhip** [comment]
I thought I only committed a resolved conflict.  ...a merge?  ???

**Comment by sjaeckel** [comment]
w/o merge there's normally no conflict :)

**Comment by karel-m** [comment]
IMO it looks more or less good, just squash it into one commit

File: src/stream/sosemanuk/sosemanuk.c [hunk 1]
```diff
@@ -755,7 +755,7 @@ int sosemanuk_crypt(sosemanuk_state *st,
 
         if (rlen > inlen) {
             rlen = inlen;
-        }
+        )
```
Comment (karel-m) [inline src/stream/sosemanuk/sosemanuk.c]: `}` was correct

**Comment by karel-m** [comment]
Another approach how to make it just a single commit:

```
git checkout develop
git pull
git checkout streams-make_state_names_consistent
git diff develop > tmp.diff
git reset develop --hard
patch -i tmp.diff -p 1
git commit -am 'rename sosemanuk_state *ss > sosemanuk_state *st'
git push --force
```

**Pull request closed without merging**